### ISSUE TITLE
What is the process for obtaining multiple mask outputs using either the model.predict() or the model.mask_decoder() method when performing instance segmentation finetuning?

### ISSUE THREAD

**Issue by Soham-Objectways** [facebookresearch/segment-anything#532]
I am seeking guidance on performing SAM fine-tuning for multi-mask instance segmentation. Additionally, during the inference stage, I wish to utilize the model.predict function to predict multiple instance segmentation masks without any prompts. This is because the actual inference images may not have any bounding boxes available.

**Comment by cristian-cmyk4** [comment]
Hi, could you find a solution? I have the same problem
